00-max.js

```diff
@@ -1,3 +1,11 @@
 export function max(array) {
-    // TODO: implement max function
+    let output;
```

Comment:
you don't need to declare your variable at the top of the function if it's not used before it's assigned for the first time.